convert/convert.go

```diff
@@ -50,6 +59,11 @@ func Convert(inputFilename, outputFilename string, from, to Format) error {
 		if err != nil {
 			return err
 		}
+	case from == FormatKongGateway2x && to == FormatKongGateway3x:
+		outputContent, err = convertKongGateway2xTo3x(inputContent)
+		if err != nil {
+			return err
+		}
```

Comment:
We need to do more. If the user doesn't supply input and output file, then all the current state files should be read and converted.

If a user's configuration is spread out over 10 files then they shouldn't be required to execute this command 10 times.